### PR TITLE
fix: do not dismiss editable comboboxes on scroll or resize immediately after opening

### PR DESCRIPTION
**Pull request by smhigley**
This is primarily geared towards fixing the issue where Android resizes the window when the keyboard opens (vs. iOS, which overlays the keyboard over the window). This means that editable comboboxes and pickers always dismiss the list of options immediately after opening on focus, since the keyboard simultaneously pops up.

## Previous Behavior

Android users at best must open a combobox or picker popup twice to view options. At worst, they can never use the popup -- this happens in the case of Talkback users, since Talkback shows/hides the keychain toolbar above the keyboard every time the popup opens and closes. Also for any picker/combobox implementations that do not provide a way to re-open the listbox popup while focus remains in the input.

## New Behavior

If the Combobox or Picker's Callout detects a scroll or resize event within 100ms of opening, it does not dismiss. This is scoped to Combobox and Picker because those are the two components that get this keyboard + Callout interaction, and limited to events within 100ms of opening, which prevents the Android keyboard dismissal but is too short for intentional user events.

## Related Issue(s)

- Fixes [16064](https://dev.azure.com/microsoftdesign/fluent-ui/_workitems/edit/16064)

### CODE REVIEW

File: change/@fluentui-react-f1224d11-1ccf-4845-805f-fe84e7b89377.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-f1224d11-1ccf-4845-805f-fe84e7b89377.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28591&commitId=6dec1229bac13851fe58c3e4f517e92b7843edd2&env=prod&clientType=FLUENTUI)


  <details><summary><b>ActivityItem</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ActivityItem.Root - RTL.default.chromium.png|695| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/f89bd4909a955569ec09396668e45e63b8faed36'>f89bd4909a955569ec09396668e45e63b8faed36</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/BaseButton-mount_base.html">610</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/BaseButton-mount.html">611</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Breadcrumb-mount_base.html">1672</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Breadcrumb-mount.html">1668</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Checkbox-mount_base.html">1655</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Checkbox-mount.html">1685</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/CheckboxBase-mount_base.html">1463</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/CheckboxBase-mount.html">1437</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/ChoiceGroup-mount_base.html">2903</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/ChoiceGroup-mount.html">2922</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/ComboBox-mount_base.html">657</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/ComboBox-mount.html">635</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/CommandBar-mount_base.html">6155</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/CommandBar-mount.html">6162</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/ContextualMenu-mount_base.html">11558</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/ContextualMenu-mount.html">11571</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/DefaultButton-mount_base.html">735</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/DefaultButton-mount.html">752</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/DetailsRow-mount_base.html">2182</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/DetailsRow-mount.html">2142</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/DetailsRowFast-mount_base.html">2179</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/DetailsRowFast-mount.html">2161</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/DetailsRowNoStyles-mount_base.html">1990</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/DetailsRowNoStyles-mount.html">1953</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Dialog-mount_base.html">2754</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Dialog-mount.html">2627</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/DocumentCardTitle-mount_base.html">242</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/DocumentCardTitle-mount.html">228</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Dropdown-mount_base.html">1952</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Dropdown-mount.html">1937</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/FocusTrapZone-mount_base.html">1117</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/FocusTrapZone-mount.html">1120</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/FocusZone-mount_base.html">1071</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/FocusZone-mount.html">1032</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/GroupedList-mount_base.html">40963</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/GroupedList-mount.html">40884</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/GroupedList-virtual-rerender_base.html">19662</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/GroupedList-virtual-rerender.html">19616</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">53397</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/GroupedList-virtual-rerender-with-unmount.html">53071</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/GroupedListV2-mount_base.html">222</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/GroupedListV2-mount.html">226</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/GroupedListV2-virtual-rerender_base.html">204</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/GroupedListV2-virtual-rerender.html">205</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">230</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">231</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/IconButton-mount_base.html">1073</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/IconButton-mount.html">1068</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Label-mount_base.html">336</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Label-mount.html">350</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Layer-mount_base.html">2684</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Layer-mount.html">2696</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Link-mount_base.html">384</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Link-mount.html">388</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/MenuButton-mount_base.html">935</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/MenuButton-mount.html">905</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/MessageBar-mount_base.html">21242</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/MessageBar-mount.html">21152</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Nav-mount_base.html">1912</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Nav-mount.html">1897</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/OverflowSet-mount_base.html">775</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/OverflowSet-mount.html">766</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Panel-mount_base.html">1757</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Panel-mount.html">1738</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Persona-mount_base.html">716</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Persona-mount.html">751</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Pivot-mount_base.html">852</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Pivot-mount.html">857</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/PrimaryButton-mount_base.html">847</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/PrimaryButton-mount.html">850</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Rating-mount_base.html">4512</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Rating-mount.html">4558</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/SearchBox-mount_base.html">903</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/SearchBox-mount.html">920</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Shimmer-mount_base.html">1841</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Shimmer-mount.html">1869</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Slider-mount_base.html">1281</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Slider-mount.html">1299</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/SpinButton-mount_base.html">2834</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/SpinButton-mount.html">2844</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Spinner-mount_base.html">385</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Spinner-mount.html">374</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/SplitButton-mount_base.html">1764</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/SplitButton-mount.html">1821</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Stack-mount_base.html">392</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Stack-mount.html">407</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/StackWithIntrinsicChildren-mount_base.html">880</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/StackWithIntrinsicChildren-mount.html">831</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/StackWithTextChildren-mount_base.html">2609</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/StackWithTextChildren-mount.html">2632</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/SwatchColorPicker-mount_base.html">6017</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/SwatchColorPicker-mount.html">5977</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/TagPicker-mount_base.html">1457</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/TagPicker-mount.html">1434</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Text-mount_base.html">364</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Text-mount.html">373</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/TextField-mount_base.html">917</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/TextField-mount.html">912</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/ThemeProvider-mount_base.html">819</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/ThemeProvider-mount.html">818</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/ThemeProvider-virtual-rerender_base.html">578</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/ThemeProvider-virtual-rerender.html">591</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1261</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1246</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Toggle-mount_base.html">603</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/Toggle-mount.html">606</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/buttonNative-mount_base.html">190</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28591/perf-test/buttonNative-mount.html">191</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Review by behowell — APPROVED** [review]

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Pull request merged**